README.md

```diff
@@ -1,2 +1,260 @@
+
+### Sierpinski Tetraheadron
+
+![gif](./media/sierpinski_tetrahedron.gif)
+
+📌 This fractal is the 3-dimensional analogue to the Sierpinski Triangle. It is also known as the Tetrix.
+
+➕ It has fractal (Hausdorff) dimension of:
+
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;D=\log_{2}{4}=2"/>
+
+🎨 The construction process for a Tetrix is very similar to that of the Sierpinski Triangle:
+1. Start with a tetraheadron of side-length L.
+2. Shrink it to half its original height.
+3. Put together four copies of the half-height tetraheadron, so that each of them touch each other in one vertex.
+4. Repeat steps 2 and 3 for the new shapes each iteration.
+
+The first iteration of this loop will give you a level-1 tetraheadron, the second one will give you a level-2 tetraheadron, and so on. The Sierpinski Tetraheadron itself is the resulting shape after infinite iterations.
+
+A similar fractal can be constructed using a square-based pyramid as its base.
+
+### Menger Sponge
+
+![gif](./media/menger_sponge.gif)
+
+📌 This fractal is a three-dimensional generalization of the two-dimensional Sierpinski Carpet, and, likewise, also a generalization of the one-dimensional Cantor Set. It was first described by Karl Menger in 1925, during his studies of topological dimnesnions.
+
+➕ It has fractal (Hausdorff) dimension of:
+
+<img src="https://latex.codecogs.com/svg.latex?\Large&space;D=\log_{3}{20}=2,7268..."/>
+
+🎨 You can construct the Menger Sponge with the following steps:
+1. Start with a cube.
+2. Subdivide each face of the cube in 9 equal sized suqares, like in a Rubik's Cube. This will subdivide the cube in 27 smaller cubes.
+3. Remove the smaller cubes on the center of each face, and the smaller cube on the very center of the larger cube (leaving a total of 20 cubes).
+4. Now, repeat steps 2 and 3 for each smaller cube infinite times an you have the fractal!
+
+The first iteration of this loop gives you a level-1 sponge, the second one gives you a level-2 sponge, and so on. The Menger Sponge itself is the resulting shape after infinite iterations.
+
+### Menger Brocolli
+
+![gif](./media/menger_brocolli.gif)
+
+📌 This is not a canonical fractal. It was constructed by me while I played with the algorithm I used to render the Menger Sponge and it is not based on any canonical fractal. I named it Menger Brocolli due to the fact that it uses the same algorithm I used for the Menger Sponge, and it also resembles a coliflower or a brocolli.\
+Unfortunately, the original algorithm I used to make that animaiton was lost, and the one provided on this repository is a mere aproximation (I couldn't find the original paramenters I used to make it like in the video).
+
+🎨 The construction of this fractal will be explained based on the algorithm used, since there is no canonical definition to its shape. It was constructed using a ray-marched Signed Distance Field (SDF) and several other space-altering functinos provided on the shader.\
```

Comment:
```suggestion
### Menger Broccoli

![gif](./media/menger_brocolli.gif)

📌 This is not a canonical fractal. It was constructed by me while I played with the algorithm I used to render the Menger Sponge and it is not based on any canonical fractal. I named it Menger Broccoli due to the fact that it uses the same algorithm I used for the Menger Sponge, and it also resembles a cauliflower or a broccoli.\
Unfortunately, the original algorithm I used to make that animation was lost, and the one provided on this repository is a mere approximation (I couldn't find the original parameters I used to make it like in the video).

🎨 The construction of this fractal will be explained based on the algorithm used, since there is no canonical definition to its shape. It was constructed using a ray-marched Signed Distance Field (SDF) and several other space-altering functions provided on the shader.\
```